modules/custom/social_advanced_queue/social_advanced_queue.install

```diff
@@ -0,0 +1,74 @@
+<?php
+
+/**
+ * @file
+ * Install and update hooks for Social Swiftmailer.
+ */
+
+/**
+ * Implements hook_install().
+ *
+ * Perform actions related to the installation of social_advanced_queue.
+ */
+function social_advanced_queue_install() {
```

Comment:
It would also be a good idea to explain or add some documentation, of why we did this? Or, why we chose this path instead of providing yml files in config/install.